C-Plus-Plus/README.md

```diff
@@ -439,6 +439,7 @@ _add list here_
 - [Reverse Bits](Bit--Manipulation/reverse_bits.cpp)
 - [Sort integers by the number of set bits present in them](Bit--Manipulation/sort_integers_by_their_number_of_set_bits.cpp)
 - [Swap_even_and_odd_bits](Bit--Manipulation/Swap_even_and_odd_bits.cpp)
+- [Find duplicate elements](Bit--Manipulation/Find_duplicate_elements.cpp)
```

Comment:
Rewrite only this part,
` [Find duplicate elements]` to  `[Find duplicate elements using XOR]`